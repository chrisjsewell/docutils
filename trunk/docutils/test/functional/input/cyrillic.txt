Заголовок
---------

первый пример: "Здравствуй, мир!"

Title
-----

.. class:: language-en

first example: "Hello world".

