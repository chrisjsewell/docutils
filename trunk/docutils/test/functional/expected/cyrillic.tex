\documentclass[a4paper,russian]{article}
% generated by Docutils <http://docutils.sourceforge.net/>
\usepackage{fixltx2e} % LaTeX patches, \textsubscript
\usepackage{cmap} % fix search and cut-and-paste in Acrobat
\usepackage{ifthen}
\usepackage[english,russian]{babel}
\usepackage[T1,T2A]{fontenc}
\usepackage[utf8]{inputenc}

%%% Custom LaTeX preamble
% PDF Standard Fonts
\usepackage{mathptmx} % Times
\usepackage[scaled=.90]{helvet}
\usepackage{courier}

%%% User specified packages and stylesheets
\usepackage{cmlgc}

%%% Fallback definitions for Docutils-specific commands

% hyperlinks:
\ifthenelse{\isundefined{\hypersetup}}{
  \usepackage[colorlinks=true,linkcolor=blue,urlcolor=blue,unicode]{hyperref}
  \urlstyle{same} % normal text font (alternatives: tt, rm, sf)
}{}


%%% Body
\begin{document}


%___________________________________________________________________________

\section*{Заголовок%
  \phantomsection%
  \addcontentsline{toc}{section}{Заголовок}%
  \label{id1}%
}

первый пример: \glqq{}Здравствуй, мир!\grqq{}


%___________________________________________________________________________

\section*{Title%
  \phantomsection%
  \addcontentsline{toc}{section}{Title}%
  \label{title}%
}

{\selectlanguage{english}first example: \glqq{}Hello world\grqq{}.}

\end{document}
